workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/AndroidRenderWorkflow.kt

```diff
@@ -0,0 +1,266 @@
+package com.squareup.workflow1.ui
+
+import androidx.lifecycle.SavedStateHandle
+import com.squareup.workflow1.ExperimentalWorkflowApi
+import com.squareup.workflow1.Workflow
+import com.squareup.workflow1.WorkflowInterceptor
+import com.squareup.workflow1.renderWorkflowIn
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.SharingStarted.Companion.Eagerly
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.onEach
+import kotlinx.coroutines.flow.stateIn
+
+/**
+ * An Android `ViewModel`-friendly wrapper for [com.squareup.workflow1.renderWorkflowIn],
+ * for use with a [workflow] that takes no input (that is, has `PropsT` set to [Unit]).
+ *
+ *    @OptIn(WorkflowUiExperimentalApi::class)
+ *    class HelloWorkflowActivity : AppCompatActivity() {
+ *      override fun onCreate(savedInstanceState: Bundle?) {
+ *        super.onCreate(savedInstanceState)
+ *
+ *        val model: HelloViewModel by viewModels()
```

Comment:
Did you mean to put this in the method instead at the class level?

Reply:
Pretty sure it won't work there.